style.css

```diff
@@ -0,0 +1,671 @@
+  color: #091e42;
+  margin-bottom: 15px;
+}
+
+.about-myself .about-content p {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  margin-bottom: 15px;
+}
+
+.about-myself .about-content h3 {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  margin-bottom: 15px;
+}
+
+.about-myself .about-content ul {
+  display: flex;
+  margin-bottom: 25px;
+}
+
+.about-myself .about-content ul li {
+  margin-right: 25px;
+}
+
+.about-myself .about-content button {
+  padding: 5px 10px;
+  border-radius: 7px;
+  border: 1px solid #396df2;
+  transition: all 1s;
+}
+
+.contact-container form .btn button {
+  padding: 10px 10px;
+  border-radius: 5px;
+  border: none;
+}
+
+.about-myself .about-content button:hover {
+  background-color: #396df2;
+}
+
+.contact-container form .btn button a {
+  color: #6070ff;
+}
+
+.about-myself .about-content button a:hover {
+  color: #fff;
+}
+
+.about-myself .about-side {
+  margin-left: 30px;
+}
+
+.about-myself .about-side ul .list1 {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+}
+
+.about-myself .about-side ul .list {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+  border-bottom: 1px solid #dfe1e6;
+}
+
+.about-myself .about-side ul .list1 .pa {
+  margin-right: 400px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.about-myself .about-side ul .list .pa {
+  margin-right: 400px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.about-myself .about-side .icon-list {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+.about-myself .about-side .icon-list li {
+  width: 122px;
+  height: 120px;
+  padding-top: 25px;
+  background: #f7f7f9;
+}
+
+.contact-container .para p {
+  width: 400px;
+  text-align: center;
+}
+
+.about-myself .about-side .icon-list li p {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
+
+@media (max-width: 500px) {
+  .about-myself .about-background {
+    width: 327px;
+    display: inline;
+  }
+
+  .about-myself .about-content {
+    width: 327px;
+    margin-bottom: 50px;
+  }
+
+  .about-myself .about-side {
+    margin-left: 0;
+  }
+
+  .about-myself .about-side ul .list1 .pa {
+    margin-right: 200px;
+  }
+
+  .about-myself .about-side ul .list .pa {
+    margin-right: 200px;
+  }
+
+  .about-myself .about-side .icon-list {
+    display: inline;
+  }
+
+  .about-myself .about-side .icon-list li {
+    display: flex;
+    align-items: center;
+    height: auto;
+    width: 327px;
+    padding: 10px;
+    margin-bottom: 20px;
+  }
+
+  .about-myself .about-side .icon-list li img {
+    margin-right: 20px;
+  }
+}
+
+/* This is the contact me area */
+
+.contact {
+  background-color: #fff;
+}
+
+.contact-container {
+  padding-top: 100px;
+  padding-bottom: 100px;
+  background-color: #6070ff;
+  border-radius: 100px 0 0 0;
+}
```

Comment:
- [ ] Great job implementing the color changes requested by the previous reviewer! 🎊 👏🏼 However, the background image for this section is still missing for both mobile and desktop versions. Please, consider adding both to match the Figma template 🎨 as much as possible! 🤓 

Mobile:
|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/201200241-68198803-8a4e-4100-bfef-3af27cdcd872.png) | ![image](https://user-images.githubusercontent.com/77069011/201199965-f2895a4c-006d-4a08-9585-730e6dec04d9.png) |

Desktop
|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/201200691-f9b2988b-8432-4918-a852-8ebac488bdfe.png) | ![image](https://user-images.githubusercontent.com/77069011/201200454-304ad386-c08a-435c-b939-acdc2df14a3d.png) |

Please, consider adding them! 🤓 🛠️